src/SFA.DAS.Provider.PR.Web.UnitTests/Controllers/CheckDetailsController/CheckDetailsControllerGetTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.Provider.PR.Web.Constants;
+using SFA.DAS.Provider.PR.Web.Infrastructure;
+using SFA.DAS.Provider.PR.Web.Infrastructure.Services;
+using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+using SFA.DAS.Provider.PR.Web.Models.Session;
+using SFA.DAS.Provider.PR_Web.UnitTests.TestHelpers;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Provider.PR_Web.UnitTests.Controllers.CheckDetailsController;
+public class CheckDetailsControllerGetTests
+{
+    private static readonly string CancelLink = Guid.NewGuid().ToString();
+
+    [Test, MoqAutoData]
+    public void Get_BuildsExpectedViewModel(int ukprn, AddEmployerSessionModel addEmployerSessionModel)
```

Comment:
I can do that with the dependencies, but if I add in the controller, I get an issue when I try and inject in the dependencies.  "AutoFixture was unable to create an instance from Microsoft.AspNetCore.Mvc.ModelBinding.BindingInfo because creation unexpectedly failed".  I don't understand why we'd want the controller added to the parameters, as we're interested in working with the dependencies to be able to test the controller.

I will do it with the dependencies for now, and we'll talk

Reply:
ok got it now